files/default/solr-import-backup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: $0 <archive path>" >&2
+    exit 1
+fi
+
+. `dirname $0`/solr-env.sh
+
+SOLR_DIR=/var/solr/data/$CORE_NAME/data
+ARCHIVE_NAME=$1
+
+sudo systemctl stop solr
+sudo -u solr rm $SOLR_DIR/index.properties
+sudo -u solr mkdir $SOLR_DIR/index
+sudo -u solr tar -C $SOLR_DIR/index -xvzf $ARCHIVE_NAME
+sudo systemctl start solr
```

Comment:
hmm, should we throw in a 'test' solr before start?

Reply:
Snapshots are tested upon creation and only sent to S3 if they pass.